src/css/tooltip.css

```diff
@@ -1,11 +1,7 @@
 .tooltip {
     position: relative;
     display: inline-block;
-    border-bottom: 1px dotted #274247;
```

Comment:
Why was the dotted under removed? I thought that this was a good illustration that it was a tooltip to be displayed. Anyway, I´m not a fan of tooltip at all. It might be a better option to replace hoover with a question mark button. It makes it more clear and it's mobile-friendly.

Reply:
It wasn't removed. I agree it is a good illustration, I kept it in local style, could not managed to keep it in CSS file. 😞